test/unit/player.test.js

```diff
@@ -1148,7 +1149,18 @@ QUnit.test('play promise should resolve to native value if returned', function(a
   player.tech_.play = () => 'foo';
   const p = player.play();
 
-  assert.equal(p, 'foo', 'play returns foo');
+  const finish = (v) => {
```

Comment:
now that play will always return a promise if it is supported we have to use `then` to check the resolved value.